src/components/NewBeer.js

```diff
@@ -0,0 +1,70 @@
+import './NewBeer.css';
+import { useState } from 'react';
+import axios from 'axios';
+import { Form } from 'react-router-dom';
+
+const NewBeer = ({random}) => {
+
+    const baseUrl = "https://ih-beers-api2.herokuapp.com/beers";
+    const [inputs, setInputs] = useState({});
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        axios
+            .post(`${baseUrl}/new`, inputs)
+            .then(response => {
+                console.log(response)
+                setInputs({});
+            })
+            // .catch(e => console.log(e))
+            .catch(console.log)
+    }
+
+    const handleChange = (e) => {
+        setInputs({...inputs, [e.target.name]: e.target.value})
+    }
+
+
+    return <div className="NewBeer">
+        <form onSubmit={handleSubmit}>
+            <label>
+                Name: 
+                <input type="text" name="name" value={inputs.name || ""} onChange={handleChange}></input>
```

Comment:
Is not clear from the lab, but that one is the only required, so it could be good to add some kind of validation here

```suggestion
                <input type="text" name="name" value={inputs.name || ""} onChange={handleChange} required></input>
```